src/redux/categories/categories.js

```diff
@@ -0,0 +1 @@
+// categories.js
```

Comment:
- [ ] Kindly create the categories reducer and ensure the pattern given, it has action types that follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux)follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux) and the initial array is an empty array.